### ISSUE TITLE
Is the printed images among the training dataset?

### ISSUE THREAD

**Issue by sookim-ai** [nilboy/pixel-recursive-super-resolution#14]
Hi Nilboy,

Thanks a lot for sharing the code, and I am using it for my research now. I appreciate for your effort so much.

I have a quick question about printed out images from sample function.
It seems you pulled out imageset from train.shuffle_batch in DataSet class, and print them out every N iterations. Can I assume that the printed image is from training set so? Which neural net already used for training at some point? (I am curious that you did divide training set and test set separately. I guess no. But in case my ignorance in tensor flow in general may disguised myself)

Thank you,
Soo
